internal/cli/gitconfigs.go

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2021 - 2023 SUSE LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	gGitconfigAllFlag bool
+)
+
+// CmdGitconfig implements the command: epinio gitconfig
+var CmdGitconfig = &cobra.Command{
+	Use:           "gitconfig",
+	Aliases:       []string{"gitconfigs"},
+	Short:         "Epinio-controlled git configurations",
```

Comment:
Not sure what "Epinio-controlled" means. Can we rephrase this?

Something like "manage git configurations", or looking at the other commands maybe "Epinio git configurations management"

```
  app           Epinio application features
  client-sync   Downloads a client binary matching the currently logged server
  completion    Generate completion script for a shell
  configuration Epinio configuration features
  gitconfig     Epinio-controlled git configurations
  help          Help about any command
  info          Shows information about the Epinio environment
  login         Epinio login to the server
  logout        Epinio logout from server
  namespace     Epinio-controlled namespaces
  push          Push an application declared in the specified manifest
  server        Starts the Epinio server.
  service       Epinio service management
  settings      Epinio settings management
  target        Targets an epinio-controlled namespace.
  version       Print the version number
```

Probably we need a coherent way for this around all the commands.

Reply:
Done with commit 15297132f4d43459d0151bf8e8ca299b399ddb26